sensor/kubernetes/main.go

```diff
@@ -46,6 +47,7 @@ func main() {
 	} else {
 		sharedClientInterface = client.MustCreateInterface()
 	}
+	clientconn.SetUserAgent("sensor-k8s")
```

Comment:
Not simply just sensor? I don't know historically why there's this `kubernetes` directory, but the deployed component itself will be called `sensor`, so might as well just use it?